docs/cip/README.md

```diff
@@ -72,6 +72,80 @@ Ouroboros Praos cannot support the high transaction volume needed to generate th
 > 
 > - [ ] Write this section after the details of the recommended variant of Full Leios have been settled.
 
+### Ledger design challenges and properties
+
+#### Leios protocol overview
+
+To understand the ledger design challenges, it's essential first to understand the key components of the Leios protocol:
+
+**Five-Stage Pipeline**: The protocol operates through a structured five-stage pipeline: (1) **Propose** - IBs generated uniformly, (2) **Deliver 1** - IB diffusion using freshest-first strategy, (3) **Deliver 2** - complete IB delivery, (4) **Endorse** - EBs generated at stage start, (5) **Vote** - voting on EBs and certificate creation. This pipeline approach enables concurrent transaction processing while maintaining deterministic ordering and conflict resolution.
```

Comment:
```suggestion
**Five-Stage Pipeline**: The protocol operates through a structured five-stage pipeline:
1. **Propose** - IBs generated uniformly
2. **Deliver 1** - IB diffusion using freshest-first strategy
3. **Deliver 2** - complete IB delivery
4. **Endorse** - EBs generated at stage start
5. **Vote** - voting on EBs and certificate creation
```
I guess you are going for high info density here but the linear aspect of this pipeline is diminished when stuffed into a text paragraph.